src/partitioned_data/partitioned_data_pqn.jl

```diff
@@ -232,9 +232,16 @@ function build_PartitionedDataTRPQN(
   py = similar(pg)
   ps = similar(pg)
 
+  # convex_expr_tree = map(convexity_status -> is_convex(convexity_status), convexity_wrapper)
```

Comment:
remove dead code

Reply:
> Thanks. What does PCS stand for?

partitioned convex specified

Reply:
Could you please explain what those acronyms mean somewhere in the documentation?